sdk/hatchery/hatchery.go

```diff
@@ -332,6 +332,17 @@ func canRunJobWithModel(ctx context.Context, h InterfaceWithModels, j workerStar
 		return false
 	}
 
+	if containsModelRequirement && model.Disabled {
```

Comment:
I don't think that here you can have a disable model, because most of hatcheries are using /worker/model/enabled endpoint to get the models list and it excludes disabled models.